src/utils.ts

```diff
@@ -2,9 +2,9 @@ import * as tsc from 'typescript';
 import * as path from 'path';
 import * as fs from 'fs';
 import * as tsconfig from 'tsconfig';
-
-const normalize = require('jest-config').normalize;
+import {normalize} from 'jest-config';
 const setFromArgv = require('jest-config/build/setFromArgv');
+// import * as setFromArgv from 'jest-config/build/setfromArgv';
```

Comment:
Strange I seem to have tried importing it in es6 as a default import as well, but I didn't get it to work. Leaving it as a require is fine for me.